_includes/why_d4a.md

```diff
@@ -0,0 +1,113 @@
+user experience. We believe strongly that additional complexity must be
+justified by substantial additional benefit.  Some solutions are very complex to
+operate or layer unneeded additional APIs on top of Docker, but Docker for
+{{cloudprovider}} provides a Docker-native solution.
+
+Instead of having to learn one or more layers on top of Docker, with Docker for
+{{cloudprovider}} you will interact with Docker (including native Docker
+orchestration) directly.  These setups free you to focus on the thing that
+matters most: running your workloads. This will help you and your team to
+deliver more value to the business faster, to speak one common "language", and
+to have fewer details to keep in your head at once.
+
+The skills that you and your team have already learned, and will continue to
+learn, using Docker on the desktop or elsewhere will automatically carry over to
+using Docker on {{cloudprovider}} and vice versa.  The added consistency across
+clouds also helps to ensure that a migration or multi-cloud strategy would be
+easier to accomplish in the future if desired.
+
+## Skip the Boilerplate and Maintenance Work
+
+Docker for {{cloudprovider}} bootstraps all of the recommended infrastructure to
+start using Docker on {{cloudprovider}} automatically.  You won't need to worry
+about rolling your own instances, security groups, load balancers, and so on
+when using Docker for {{cloudprovider}}.
+
+Likewise, the setup of the Docker swarm mode functionality (vital for container
+orchestration) has to be instantiated when the cluster starts up and
+subsequently managed across its lifecycle.  With Docker for {{cloudprovider}},
+this is all done for you.  We have already coordinated the various bits of
+automation you would otherwise be gluing together on your own to bootstrap
+Docker swarm mode on these platforms.  When the cluster is finished booting, you
+can jump right in and start running `docker service` commands.
+
+We also provide a prescriptive upgrade path that is intended to help users
+upgrade between various versions of Docker for {{cloudprovider}} and the Docker
+Engine itself in a smooth and automatic way.  Instead of having "maintenance
+dread" set in as you ponder your future responsibilities upgrading the software
+you are using, these product will allow you to quickly and simply upgrade to new
+versions when they are released.
+
+## Minimal, Docker-Focused Base
+
+The custom Linux distribution Docker for {{cloudprovider}} is built on top of is
+painstakingly configured and developed to run Docker in the most ideal manner
+possible.  Everything from the kernel configuration to the networking stack has
+been customized to make it a favorable place for running Docker.  For instance,
+we have ensured that the kernel versions are compatible with the latest and
+greatest in Docker functionality, such as the `overlay2` storage driver.
+
+Instead of facing the tradeoffs that a general purpose operating system makes,
```

Comment:
the trade-offs of a general-purpose operating system, Docker's custom Linux distribution focuses on only one thing: providing the best _Docker_ experience for you and your team.